core/tests/data/dummy_ci_tests/.dummy_travis_fail.yml

```diff
@@ -0,0 +1,16 @@
+env:
+  jobs:
+    - RUN_E2E_TESTS_ACCESSIBILITY=true
```

Comment:
As above, use fake names that are formatted similarly to the real ones. In addition, see if you can communicate through naming why this file should fail (you could name things _IS_NOT_IN_YAML_FILE for example).

Reply:
?<=